src/Alex/Gamestates/MainMenu/OptionsState.cs

```diff
@@ -14,6 +14,8 @@ public OptionsState(GuiPanoramaSkyBox skyBox) : base(skyBox)
 
         protected override void OnInit(IGuiRenderer renderer)
         {
+            Body.ClearChildren();
```

Comment:
Instead of clearing we should probably just move the element creation to the constructor.